editor/src/clj/editor/sprite.clj

```diff
@@ -621,10 +684,6 @@
             (default {})
             (dynamic visible (g/constantly false)))
 
-  (property attributes g/Any
-            (default [])
-            (dynamic visible (g/constantly false)))
-
```

Comment:
This was unused. We store only the override values by attribute-key in the `vertex-attribute-overrides` property.